hoki/utils/binning.py

```diff
@@ -0,0 +1,180 @@
+"""
+Routines to bin spectra.
+
+Author: Martin Glatzle
+"""
+import numpy as np
+from numba import jit
+
+
+def bin_spectra(wave, spectra, bins, edges=False):
+    """
+    Bin spectra conserving luminosity.
```

Comment:
that is key - and very interesting to me. I have a (personal - no yet in hoki) binning algorithm very different from yours, because it is designed to bin spectra in **counts** (and calculate associated errors). 

I am wondering if we could call this function something more specific to indicate this is specifically designed to handle luminosities (and other non-Poissonian quantities). this way when I finally add my binning algo there won't be a clash in naming.

Reply:
Hmm. I'm sure there is a better name. But I'm not clear on how your application is different or rather how it works in the first place so I'm struggling to come up with something of the cuff. I have no problem with renaming things, of course, but I think I'd first need to understand what the difference actually is.

Reply:
fair - I would recommend we then call it `bin_luminosity`, mine would then be called `bin_counts` :)
Does that work for you?

Reply:
Done.